test/utils/Defaults.sol

```diff
@@ -29,7 +29,7 @@ contract Defaults is Merkle {
     uint40 public immutable CLIFF_TIME;
     uint40 public immutable END_TIME;
     uint256 public constant ETHER_AMOUNT = 10_000 ether;
-    uint256 public constant MAX_SEGMENT_COUNT = 1000;
+    uint256 public constant MAX_COUNT = 500;
```

Comment:
I have renamed `MAX_SEGMENT_COUNT` to `MAX_COUNT`. `MAX_SEGMENT_COUNT` is not used anywhere.